### ISSUE TITLE
my fingers are not transparent

### ISSUE THREAD

**Issue by petaflot** [8VIM/8VIM#420]
**Is your feature request related to a problem? Please describe.**
I really like the concept, but the learning curve is frustrating because the finger hides quite a number of the characters ; this is even worse with layers

**Describe the solution you'd like**
an option to show the symbols (and trace) not on the bottom area of the screen, where the circle is, but higher on the screen

**Describe alternatives you've considered**
forking?

**Comment by MaethorNaur** [comment]
Hi, we are aware of that and looking for an alternative.
- create a tutorial. (especially if the layout has more than one layer)
- having a help button with all layers and movements.

There's a guide on the Wiki (https://github.com/8VIM/8VIM/wiki/Usability-Guide) it's not great but a start

**Comment by Perdellian** [comment]
I suspect that this is a user issue. @petaflot should consider the obvious workaround of just becoming a ghost.

Seriously, though, having something like a smaller reference circle to the side might be helpful (even moreso if it mirrored your current gesture path, but I'm guessing that'd be a bit more pie-in-the sky).

**Comment by EricDru** [comment]
Transparent stylus? 
Preview letter at flashing cursor insertion point? 
#429

**Comment by supercheetah** [comment]
I just discovered this project and love it, and thought quite a bit about this issue, and I think the best way to deal with this issue is to have a little pop-up near the top of the keyboard, or hovering over the touch point that shows the current row with the current letter highlighted, like this (forgive my lack of artistry, and bad video editing skills, so use this as just inspiration, but it shouldn't look like this in the end):
![Letter row highlight current](https://github.com/8VIM/8VIM/assets/725981/06eeb6ff-f462-4bdd-94c5-77237029cb7e)

**Comment by SbAgOoTs** [comment]
Just started using 8Vim and I totally agree with the request of @petaflot : the right-down branch is never readable, because of your finger, an indication of the currently selected letter inside the circle or above the keyboard would easily solve the problem.

**Comment by Blake-LeBlanc** [comment]
What if the angle of the X could be rotated? If it were more of a +, for example, the user's thumb might no longer cover the diagonal line.

**Comment by SbAgOoTs** [comment]
Maybe that's the easiest solution indeed...

**Comment by pearagit** [comment]
> What if the angle of the X could be rotated? If it were more of a +, for example, the user's thumb might no longer cover the diagonal line.

good idea. one potential issue is there would be less space for characters along the lines, though
